independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/QualifierRegistrar.java

```diff
@@ -12,7 +12,8 @@
 
     /**
      * Returns a map of additional qualifers where the key represents the annotation type and the value is an optional set of
-     * non-binding members.
+     * non-binding members. Here, "non-binding" is meant in the sense of `javax.enterprise.util.Nonbinding`. I.e. members named
```

Comment:
This is javadoc, we should probably use something like `{@link javax.enterprise.util.Nonbinding}` or `{@code javax.enterprise.util.Nonbinding}` instead...

Reply:
+1, sorry for selecting a bad serializer :)

Reply:
Done in dd58052